internal_tests/pytests/gen_vx_mask/test_gen_vx_mask.py

```diff
@@ -0,0 +1,135 @@
+
+#@pytest.fixture
+def metplus_config():
+    """! Create a METplus configuration object that can be
+    manipulated/modified to
+         reflect different paths, directories, values, etc. for individual
+         tests.
+    """
+    try:
+        if 'JLOGFILE' in os.environ:
+            produtil.setup.setup(send_dbn=False, jobname='GenVxMaskWrapper',
+                                 jlogfile=os.environ['JLOGFILE'])
+        else:
+            produtil.setup.setup(send_dbn=False, jobname='GenVxMaskWrapper')
+        produtil.log.postmsg('gen_vx_mask_wrapper  is starting')
+
+        # Read in the configuration object CONFIG
+        config = config_metplus.setup(util.baseinputconfs)
+        logger = util.get_logger(config)
+        return config
+
+    except Exception as e:
+        produtil.log.jlogger.critical(
+            'gen_vx_mask_wrapper failed: %s' % (str(e),), exc_info=True)
+        sys.exit(2)
+
+
+# ------------------------ TESTS GO HERE --------------------------
+
+
+# ------------------------
+#  test_
+# ------------------------
+def test_run_gen_vx_mask_once():
+    input_dict = {'valid': datetime.datetime.strptime("201802010000",'%Y%m%d%H%M'),
+                  'lead': 0}
+    time_info = time_util.ti_calculate(input_dict)
+
+    wrap = gen_vx_mask_wrapper()
+    wrap.c_dict['INPUT_TEMPLATE'] = '{valid?fmt=%Y%m%d%H}_ZENITH'
+    wrap.c_dict['MASK_INPUT_TEMPLATES'] = ['LAT']
+    wrap.c_dict['OUTPUT_DIR'] = os.path.join(wrap.config.getdir('OUTPUT_BASE'),
+                                             'GenVxMask_test')
+    wrap.c_dict['OUTPUT_TEMPLATE'] = '{valid?fmt=%Y%m%d%H}_ZENITH_LAT_MASK.nc'
+    wrap.c_dict['COMMAND_OPTIONS'] = ["-type lat -thresh 'ge30&&le50'"]
+#    wrap.c_dict['MASK_INPUT_TEMPLATES'] = ['LAT', 'LON']
+#    wrap.c_dict['COMMAND_OPTIONS'] = ["-type lat -thresh 'ge30&&le50'", "-type lon -thresh 'le-70&&ge-130' -intersection"]
+
+    wrap.run_at_time_all(time_info)
+
+    expected_cmd = f"{wrap.app_path} 2018020100_ZENITH LAT {wrap.config.getdir('OUTPUT_BASE')}/GenVxMask_test/2018020100_ZENITH_LAT_MASK.nc -type lat -thresh 'ge30&&le50' -v 2"
+
+    for cmd in wrap.all_commands:
+        print(f"COMMAND:{cmd}")
+        print("EXPECTED:{expected_cmd}")
+        assert(cmd == expected_cmd)
+
+def test_run_gen_vx_mask_twice():
+    input_dict = {'valid': datetime.datetime.strptime("201802010000",'%Y%m%d%H%M'),
+                  'lead': 0}
+    time_info = time_util.ti_calculate(input_dict)
+
+    wrap = gen_vx_mask_wrapper()
+    wrap.c_dict['INPUT_TEMPLATE'] = '{valid?fmt=%Y%m%d%H}_ZENITH'
+    wrap.c_dict['MASK_INPUT_TEMPLATES'] = ['LAT', 'LON']
+    wrap.c_dict['OUTPUT_DIR'] = os.path.join(wrap.config.getdir('OUTPUT_BASE'),
+                                             'GenVxMask_test')
+    wrap.c_dict['OUTPUT_TEMPLATE'] = '{valid?fmt=%Y%m%d%H}_ZENITH_LAT_LON_MASK.nc'
+    cmd_args = ["-type lat -thresh 'ge30&&le50'", "-type lon -thresh 'le-70&&ge-130' -intersection"]
```

Comment:
Recommend adding "-name lat_lon_mask" to the 2nd cmd_args entry to define the output variable name:cmd_args = ["-type lat -thresh 'ge30&&le50'", "-type lon -thresh 'le-70&&ge-130' -intersection -name lat_lon_mask"]